### ISSUE TITLE
Segmentation Fault

### ISSUE THREAD

**Issue by wiperpaul** [noelmartinon/mboxzilla#1]
I came across the following issue when dealing with a large mbox file and attempting to parse it before trying to -e extract and -s split. This results in about 700 of the 2000 eml files being outputted. The mbox file I am using are known phishing emails and include a lot of "fun" encoding, could this be the issue ?

( how could i produce more logging to review this ?)

````
mboxzilla version 1.1.0
Copyright (C) 2017-2019 Noel Martinon. All rights reserved.

2019-07-01 14:41:01,766 INFO STARTING mboxzilla
2019-07-01 14:41:01,766 INFO INPUT FILE is "emails.mbox"
2019-07-01 14:41:01,766 INFO OUTPUT DIRECTORY is "emlfiles/"
2019-07-01 14:41:01,767 INFO Start parsing file "emails.mbox"
Segmentation fault=                                ]  36% /
```

**Comment by noelmartinon** [comment]
Hello
Are you ok to send to me a part of your 'emails.mbox' ?

**Comment by noelmartinon** [comment]
The encoding is certainly the problem. 
You should have more informations by extracting with verbose 3 option, get the datetime of the last exported mail and check manually in the mbox file what's wrong after this one:
`./mboxzilla -f "emails.mbox" -o "eml_export" -v 3 -c`
I think that the next message is not found. Begiining with '\nFrom '.

**Comment by wiperpaul** [comment]
Hi noel, this is the mbox file
https://github.com/diegoocampoh/MachineLearningPhishing/blob/master/code/resources/emails-phishing.mbox

**Comment by noelmartinon** [comment]
Thank you for your file and trusting me. The issue was a malformed date '07:08:2006' !
I released a new version  (1.2.0).
And now, after exporting your mbox file, open the incriminated mail 20060807220318_b4480c8bf58a3965f3665aface5914d7.eml with an email client and check its date ... in Thunderbird there is none displayed!
